view/main/index.pug

```diff
@@ -1,4 +1,4 @@
-main.main
+main.layout_main.main
```

Comment:
BEM isolation leak again. When you are writing `.layout_main` styles, you need to keep in mind `.main` styles.

On any `.main` styles changes you can make a conflict with `.layout_main`.

Reply:
All layout_n classes are about the location of the blocks themselves. There is no need for context here. Nothing else will be added to this class

According to BEM, a block should not have any external geometry (position absolute , margin, width/height, border, etc.)

Based on this, how will layout_n affect the block itself?

I mean, we don’t write anything from external geometry to the block. If we need the outer geometry itself, then we will use the mix and give it (width / height, positioning and so on)

Reply:
> If we need the outer geometry itself, then we will use the mix and give it (width / height, positioning and so on)

We can add a `<div>` wrapper (element) to add external geometry. I prefer this way.

Reply:
I understand you

For me, an additional class is better than a div. The DOM itself is smaller and generally more logical. Each wrapper should be created only if something cannot be done functionally or if it helps to create a landmark or something else for screen reader

I look at the styles of the `card` component and see that it says fixed width. According to BEM, you can't do that. You cannot write a fixed width to a component, because it can be reused elsewhere. The best solution is to create a conditional `layout_card` mix and give a width to this class to isolate the external geometry

Can I send a new pull request that will solve these issues or how? Will it be better for BEM?

Close this PR?